src/vm/dllimport.cpp

```diff
@@ -2950,6 +2960,150 @@ void PInvokeStaticSigInfo::DllImportInit(MethodDesc* pMD, LPCUTF8 *ppLibName, LP
     }
 }
 
+#if !defined(CROSSGEN_COMPILE) // IJW
+
+#if defined (FEATURE_PAL)
+#pragma clang diagnostic push
```

Comment:
I would be better (fewer lines, less ceremony) to put the whole function under `#if !defined(FEATURE_PAL)` instead of disabling the warnings.

Reply:
Done. (This didn't work anyway).